frontend/src/host_orchestrator/api/v1/messages.go

```diff
@@ -208,3 +208,5 @@ type DisplayScreenshotResponse struct {
 	ScreenshotBytesBase64 string `json:"screenshot_bytes"`
 	ScreenshotMimeType    string `json:"screenshot_mime_type"`
 }
+
+type ScreenRecordingListResponse []string
```

Comment:
use name `ListScreenRecordingResponse`. I noticed the naming pattern was not followed with Display* object names, I'll send a PR fixing that, however let's keep the original pattern with the new screen recording methods.